sw/host/opentitanlib/src/transport/verilator/subprocess.rs

```diff
@@ -35,9 +35,10 @@ pub struct Subprocess {
 impl Subprocess {
     /// Starts a verilator [`Subprocess`] based on [`Options`].
     pub fn from_options(options: Options) -> Result<Self> {
-        let mut command = Command::new(&options.executable);
+        let mut command = Command::new(String::from("nice"));
         let mut args = Vec::new();
```

Comment:
Out-of-scope nit: Looks like `Command::arg()` and `Command::args()` exist. Given that `command` is already mutable, I think we could probably eliminate the `args` vector. Not super important and adds more noise to the diff, though.